ui/apps/platform/src/routePaths.ts

```diff
@@ -306,14 +318,24 @@ const routeRequirementsMap: Record<RouteKey, RouteRequirements> = {
     'vulnerabilities/platform-cves': {
         resourceAccessRequirements: everyResource(['Cluster']),
     },
-    'vulnerabilities/platform-workload-cves': {
+    'vulnerabilities/reports': {
+        resourceAccessRequirements: everyResource(['WorkflowAdministration']),
+    },
+    'vulnerabilities/user-workloads': {
```

Comment:
Am impression, for what it is worth. I searched to see if `'vulnerabilities/user-workloads'` was
* instead of `'vulnerabilities/workload-cves'` key
* in addition to `'vulnerabilities/workload-cves'` key

Eventually I expanded the unchanged lines below to find that it is in addition to. Maybe more it up here?